manager/services/certificate_signer.go

```diff
@@ -44,7 +44,7 @@ func (h HttpError) Error() string {
 	return fmt.Sprintf("http status: %d", h)
 }
 
-func (h HubjectCertificateSignerService) SignCertificate(typ CertificateType, pemEncodedCSR string) (string, error) {
+func (h CPOCertificateSignerService) SignCertificate(typ CertificateType, pemEncodedCSR string) (string, error) {
```

Comment:
So this implementation uses the Open Plug & Charge protocol (OPCP) and is specific to signing CPO certificates so would probably name it OPCPCPOCertificateSignerService

Reply:
Seeing it written down we probably want OpcpCpoCertificateSignerService as we're joining multiple acryonyms together.